docs/concepts/executor/add-endpoints.md

```diff
@@ -122,21 +83,14 @@ class MyExecutor(Executor):
 
 Let's take a look at these arguments:
 
-- `docs`: A DocumentArray that is part of the request. Since the nature of Executor is to wrap functionality related to `DocumentArray`, it's usually the main processing unit inside Executor methods. It's important to notice that these `docs` can be also changed in place, just like
-any other `list`-like object in a Python function.
-
+- `docs`: A DocumentArray that is part of the request. Since an Executor wraps functionality related to `DocumentArray`, it's usually the main data format inside Executor methods. Note that these `docs` can be also change in place, just like any other `list`-like object in a Python function.
 - `parameters`: A Dict object that passes extra parameters to Executor functions.
-
-- `docs_matrix`:  This is one of the least common parameters to be used for an Executor. It is passed when multiple parallel branches lead into the Executor, and `no_reduce=True` is set. Each DocumentArray in the matrix is the output of one previous Executor.
```

Comment:
remember to explain this in another section. How can we make sure we do not forget?